api/tsconfig.json

```diff
@@ -20,6 +20,6 @@
     ]
   },
   "include": [
-    "src/**/*.ts",
+    "src/**/*.ts", "logger.ts", "config.ts", "knexfile.ts", "src/database/migrations/*.ts",
```

Comment:
wondering why you've got wildcards for everything but the stuff in the root. Perhaps it makes more sense to move `logger.ts` and `config.ts` into `config/` dir and then use `config/**/*.ts`?